lms/djangoapps/program_enrollments/docs/decisions/0002-matriculation-on-user_social_auth_change.rst

```diff
@@ -0,0 +1,60 @@
+Matriculation on UserSocialAuth Changes
+---------------------------------------
+
+Status
+======
+
+Accepted (circa October 2019)
+
+
+Context
+=======
+
+Georgia Tech has been enrolling learners into its Master's programs
+for a long time, using enterprise-oriented APIs for the purpose. To
```

Comment:
"for a long time" -> "before the program enrollment system was built"

Reply:
I chose to ignore these accidentally! Github told me I had an approval before it loaded these comments 😬